blackbox/hdfs/src/tests.py

```diff
@@ -179,7 +179,13 @@ def test_suite():
         host='localhost',
         crate_home=crate_path(),
         port=CRATE_HTTP_PORT,
-        transport_port=CRATE_TRANSPORT_PORT
+        transport_port=CRATE_TRANSPORT_PORT,
+        settings={
+            # The disk.watermark settings can be removed once crate-python > 0.21.1 has been released
+            "cluster.routing.allocation.disk.watermark.low" : "85%",
```

Comment:
think we should use the same values as in https://github.com/crate/crate/pull/6823/files#diff-0d77e0dc656c95d990aeff214e1680e7R155